src/Body.java

```diff
@@ -1,6 +1,10 @@
+import java.util.ArrayList;
+import java.util.List;
+
 public class Body {
     private Organ[] organList;
     private int globalHP;
+    private List<ImmuneCell> reserves;
```

Comment:
Provide documentation on what reserves is, what it holds, any further conditions including spawn time of cells.